README.md

```diff
@@ -19,57 +19,30 @@ The federation_service can be installed in a py3.7+ virtual environment:
 pip install -r requirements.txt
 ```
 
-## Configuration Files
-
-The Federation Service requires two JSON configuration files to be placed into the `federation_service/configs` directory. 
-These files are `servers.json` and `services.json`.
-
-### How to register services
-
-Specify the name and URL of your services.Below is an example of `services.json`:
-
-```
-{
-  "services": {
-    "katsu": "https://katsu-api-2.herokuapp.com",
-    "candig-server": "https://app05.herokuapp.com"
-  }
-}
-```
-
 ### How to register peer servers
 
-For federation to work, you will need to register both the `host` and its peer servers as `servers` under `config/servers.json`. This may be confusing, as you may think that your `host` does not need to be registered.
-
-In addition, specify the location for your peer servers with the array `["research centre", "province name", "province code"]`. For compatibility with CanDIG's [data portal](https://github.com/CanDIG/candig-data-portal), use the following province codes:
-
-`'ca-ab', 'ca-bc', 'ca-mb', 'ca-nb', 'ca-nl', 'ca-nt', 'ca-ns', 'ca-nu', 'ca-on', 'ca-pe', 'ca-qc', 'ca-sk', 'ca-yt'`
-
-
-For example, if your host federation service is running at `http://0.0.0.0:8890` in British Columbia, and your first 
-peer server federation service is running at `http://0.0.0.0:8891` in Ontario, your `servers.json` would look like this:
+On initialization of the docker container, the server listed in .env as FEDERATION_SELF_SERVER will be registered. This is your own server. If you want to register other peer servers, use the /federation/v1/servers POST endpoint, described in federation.yaml. You'll need to have a valid JWT from the peer server's identity issuer, as well as the URL of the issuer. This call must be authorized with a bearer token from a site administrator.
```

Comment:
Can you be explicit about the two pieces of information that we need from the JWT in order to set up a new server? Also, be specific that the admin bearer token is from the local instance, not the peer being added.

Reply:
https://github.com/CanDIG/federation_service/pull/36/commits/8be405e886904bb37ad350a56bde0996849d6c6e

Is that more clear?